src/AllocatorRoles.sol

```diff
@@ -22,16 +22,16 @@ contract AllocatorRoles
     // --- storage variables ---
 
     mapping(address => uint256) public wards;
-    mapping(address => bytes32) public userRoles;
-    mapping(address => mapping(bytes4 => bytes32)) public actionsRoles;
-    mapping(address => mapping(bytes4 => uint256)) public publicActions;
+    mapping(bytes32 => address) public domains;
```

Comment:
Maybe this should be named `admins` or `domainAdmins` ?

Reply:
Done!